main.py

```diff
@@ -58,7 +45,7 @@ def get_authors_list(db: Session = Depends(get_db)):
 
 @app.get("/authors/{author_id}/", response_model=schemas.AuthorReadDetail)
 def get_author_detail(author_id: int, db: Session = Depends(get_db)):
-    if author := crud.get_author_by_id(db, author_id):
+    if author := crud.get_author_by_id(db=db, author_id=author_id):
```

Comment:
no need to pass parameters' name explicitly, just reorder them so that it is in the same order that in function